README.md

```diff
@@ -14,30 +14,32 @@ A [Singer](https://singer.io/) postgres target, for use with Singer streams gene
 
 ## Usage
 
-Create a config file with postgres connection information and target postgres schema.
-
-```json
-{
-  "postgres_host": "locahost",
-  "postgres_port": 5432,
-  "postgres_database": "my_analytics",
-  "postgres_username": "myuser",
-  "postgres_password": "1234",
-  "postgres_schema": "mytapname"
-}
-```
+1. Follow the
+   [Singer.io Best Practices](https://github.com/singer-io/getting-started/blob/master/docs/RUNNING_AND_DEVELOPING.md#running-a-singer-tap-with-a-singer-target)
+   for setting up separate `tap` and `target` virtualenvs to avoid version
+   conflicts.
 
-Run `target-postgres` against a [Singer](https://singer.io) stream.
+1. Create a config file with postgres connection information and target
+   postgres schema.
 
-```sh
-	tap-something | target-postgres --config config.json
-```
+   ```
+   cp config-example.json /tmp/tap_config.json
```

Comment:
Can we keep the example config rather than introduce a confusing set of commands?

Reply:
Also, this is a target, the file name has tap

Reply:
🤦‍♂️ I didn't even notice that the `config-example.json` file was being ignored. :)